client/src/components/Header/Header.component.jsx

```diff
@@ -110,9 +106,7 @@ const Header = () => {
   }
 
   // Look for /questions to catch search result and post pages
-  const matchQuestions = matchPath(location.pathname, {
-    path: '/questions',
-  })
+  const matchQuestions = matchPath(location.pathname, '/questions')
```

Comment:
This might not be working well - behaviour for header on questions page has changed. Will look into it.

Reply:
yea the arguments should be swapped

Reply:
oof okays thanks 👍

Reply:
Header still isn't acting normally, give me a moment